k8s/helm/templates/proxy-config.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.manager.env.proxy }}
```

Comment:
This does not work because the secret already exists and can't be added to the helm release of the blueprint-operator. For now the setup just should add the proxy and other components just use it.

In future the blueprint maybe configures or changes the proxy in the blueprint process in code.